modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -2525,25 +2529,26 @@ def to_pandas(self):
                 for idx, new_type in cast.items():
                     schema = schema.set(idx, pyarrow.field(new_type, pyarrow.string()))
                 at = at.cast(schema)
-            df = at.to_pandas()
+            df = concatenate([at.to_pandas()])
```

Comment:
please add an in-code comment on why we call `concatenate` here

Reply:
Done